cmd/dhcpserver/cmd.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 IBM Corp
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dhcpserver
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ppc64le-cloud/pvsadm/pkg"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "dhcpserver",
+	Short: "dhcpserver command",
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if pkg.Options.InstanceID == "" {
```

Comment:
Is there any reason to do like this instead of using `cmd.MarkFlagRequired` ?

Reply:
that's because these flags are coming from the parent command and we can't set the required in the child commands.

Reply:
yeah ok, but I think `--instance-id` is required in all child commands to create pvmclient I guess, if it is required in all the child commands maybe we can mark required in parent command itself I guess. What do you think?

Reply:
My bad, you are right! Written this long back and almost forgot what I have coded :P.. lemme fix it.

Reply:
removed this check and added as `Required` flag